drivers/serial/Kconfig

```diff
@@ -148,6 +155,15 @@ config UART_ASYNC_TO_INT_DRIVEN_RX_TIMEOUT
 	help
 	  Receiver inactivity timeout. It is used to calculate timeout in microseconds.
 
+config UART_BUILTIN_HALFDUPLEX
+    bool "Built-in half-duplex API"
+    depends on SERIAL_SUPPORT_BUILTIN_HALFDUPLEX
+    default n 
```

Comment:
`default n` is the default, omit this line.

Reply:
did not notice this thanks